wpe/src/main/java/com/wpe/wpe/Page.java

```diff
@@ -257,6 +257,25 @@ public void onInputMethodContextOut()
         dismissKeyboard();
     }
 
+    public void enterFullscreenMode()
+    {
+        Log.v(LOGTAG, "enterFullscreenMode");
+        m_wpeView.enterFullScreen();
+    }
+
+    public void exitFullscreenMode()
+    {
+        Log.v(LOGTAG, "exitFullscreenMode");
+        m_wpeView.exitFullScreen();
+    }
+
+    public void requestExitFullscreenMode()
+    {
+        if (m_pageGlueReady) {
+            BrowserGlue.requestExitFullscreenMode(m_id);
+        }
```

Comment:
{} are not useful

Reply:
follows the convention that was already used elsewhere in the file for similar code. We can reformat whole file later with consistent convention